### PR TITLE
Integrate Mayhem

### PR DESCRIPTION
**Pull request by rnshah9**

### CODE REVIEW

File: Dockerfile.mayhem [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM --platform=linux/amd64 ubuntu:22.04 as builder
```
Comment (xansec) [inline Dockerfile.mayhem:1]: Please add a package stage here - this looks unfinished.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: Dockerfile.mayhem (hunk 1 above)
Comment (rnshah9) [reply Dockerfile.mayhem]: Basically the only dependency I could trim was Git.

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**